Dev to main

VictoriaZmushko opened the pull request:
~~Added different HTML docs. Wrote examples with styles, elements, containers, comments, etc. Practiced a little with css.~~

**Лучше так:**

1. Added different HTML docs. 
2. Wrote examples with:
  - styles, 
  - elements, 
  - containers, 
  - comments, etc. 
3. Practiced a little with css.

Check list:
- [ ] 1. Added different HTML docs. 
- [ ] 2. Wrote examples with:
- [ ]     - styles, 
- [ ]     - elements, 
- [ ]     - containers, 
- [ ]     - comments, etc. 
- [ ] 3. Practiced a little with css.

Commit:
add simple doc

Commit:
add doc with frame

Commit:
add HTML-file

Commit:
add file with icon

Commit:
add file with HTML headings

Commit:
add example link and image

Commit:
testing different tags

Commit:
add Text Formatting

Commit:
add commets

Commit:
Example list

Commit:
Example table

Commit:
add submit button

Commit:
add container

Commit:
add example

Commit:
add color

Commit:
adding containers on the same page

Commit:
add inline-block

Commit:
add example styles

Commit:
add example flex

Commit:
add example grid

Commit:
adding styles

Commit:
use tag div

Commit:
example float

Commit:
add example flexbox

udevgeektutor commented:
Added different HTML docs. Wrote examples with styles, elements, containers, comments, etc. Practiced a little with css.

**Лучше так:**

1. Added different HTML docs. 
2. Wrote examples with:
    - styles, 
    - elements, 
    - containers, 
    - comments, etc. 
3. Practiced a little with css.

Check list:
- [ ] 1. Added different HTML docs. 
- [ ] 2. Wrote examples with:
- [ ]     - styles, 
- [ ]     - elements, 
- [ ]     - containers, 
- [ ]     - comments, etc. 

- [ ] 3. Practiced a little with css.

VictoriaZmushko commented:
Added different HTML docs.
Wrote examples with:
styles,
elements,
containers,
comments, etc.
Practiced a little with css.
Check list:

 1. Added different HTML docs.
 2. Wrote examples with:
 - styles,
 - elements,
 - containers,
 - comments, etc.
 3. Practiced a little with css.
